htdocs/fourn/facture/card.php

```diff
@@ -2411,6 +2411,8 @@
 				print $langs->trans('AlreadyPaid');
 			print ' :</td><td align="right"'.(($totalpaye > 0)?' class="amountalreadypaid"':'').'>' . price($totalpaye) . '</td><td>&nbsp;</td></tr>';
 
+            $totalpaye = $object->getSommePaiement();
```

Comment:
I think The the `  $totalpaye = $object->getSommePaiement();` should be removed because we already have `$totalpaye` calculated  in the [loop](https://github.com/tarrsalah/dolibarr/blob/840408c1d21f6895c821b78c62a7bd7090c461aa/htdocs/fourn/facture/card.php#L2375) above.